src/material-table.js

```diff
@@ -182,42 +182,23 @@ export default class MaterialTable extends React.Component {
 
     calculatedProps.actions = [...(calculatedProps.actions || [])];
 
-    if (calculatedProps.options.selection)
-      calculatedProps.actions = calculatedProps.actions
-        .filter((a) => a)
```

Comment:
removed because `a` is never falsy, type of `a` is `Action<RowData> | ((rowData: RowData) => Action<RowData>)`